python/gps/algorithm/traj_opt/traj_opt_lqr_python.py

```diff
@@ -30,7 +32,19 @@ def update(self, m, algorithm):
         step_mult = algorithm.cur[m].step_mult
         prev_eta = algorithm.cur[m].eta
         traj_info = algorithm.cur[m].traj_info
-        prev_traj_distr = algorithm.cur[m].traj_distr
+
+        if type(algorithm) == AlgorithmMDGPS:
+            # For MDGPS, constrain to previous NN linearization
+            # NOTE: we only copy in necessary params, ignoring inv_pol_covar
+            pol_info = algorithm.cur[m].pol_info
```

Comment:
I was debating about that. My plan was to just add a `pol_info_to_traj_distr` function in `algorithm_utils.py`, but I decided against it because the proper thing would be to compute 'inv_pol_covar' and that's never needed. Definitely premature optimization though.. so I'll just add that helper method. In the refactor, I think we might want to add a "linearization" object which can come from a policy or LG controller. We could use that everywhere traj_distr or a neural net linearization is needed.